store.html

```diff
@@ -0,0 +1,102 @@
+    </nav>
+    <main>
+        <div class="flex-container">
+            <section class="form">                
+                <form>
+                    <h1>Order your Karma wifi device today!</h1>
+                    <div class="input-field">
+                        <div class="two-inputs">
+                            <div class="name">
+                                <label class="required" for="firstName">First Name</label>
+                                <input type="text" id="firstName" name="firstName" required />    
+                            </div>
+                            <div class="name">
+                                <label class="required" for="lastName">Last Name</label>
+                                <input type="text" id="lastName" name="lastName" required />
+                            </div>                        
+                        </div>                        
+                        <label class="required" for="address">Address</label>
+                        <input type="text" id="address" name="address" required>
+                        <label for="address2">Address 2</label>
+                        <input type="text" id="address2" name="address2">
+                       <div class="two-inputs">
+                           <div class="cities">
+                            <label class="required" for="city">City</label>
+                            <select id="cities" name="cities">                                                
+                            <option value="" disabled selected>Choose a city</option>                
+                              <option value="london">London</option>
+                              <option value="birmingham">Birmingham</option>                         
+                              <option value="manchester">Manchester</option>                         
+                              <option value="birmingham">Glasgow</option>                
+                              <option value="brighton">Brighton</option>                            </select>
+                           </div>
+                           <div class="postcode">
+                            <label class="required" for="postcode">Postcode</label>
+                            <input type="text" id="postcode" name="postcode" required>
+                           </div>                      
+                       </div>                  
+                    </div>
+                            <div>
+                                <legend>Select a colour</legend>    
+                                <div class="radio-buttons">                                                         
+                                    <div class="radio-button">
+                                        <input type="radio" id="karma-orange" value="karmaOrange" name="radio" checked />
+                                        <label for="karmaOrange">Karma orange</label>
```

Comment:
`for` and `id` tags should match (not `for` and `value`)

Also `radio` is not a very descriptive name -- "colour" would work better here.